controllers/clusterurlmonitor/clusterurlmonitor.go

```diff
@@ -67,16 +63,26 @@ func NewReconciler(mgr manager.Manager, blackboxExporterImage, blackboxExporterN
 	}
 }
 
+const (
+	FinalizerKey string = "clusterurlmonitor.routemonitoroperator.monitoring.openshift.io/finalizer"
+	// PrevFinalizerKey is here until migration to new key is done
+	PrevFinalizerKey string = "clusterurlmonitor.monitoring.openshift.io/clusterurlmonitorcontroller"
```

Comment:
Why do we need a different key?

Reply:
This is a part of the migration, one of the commits talks about it